goon-universe/queries.js

```diff
@@ -378,7 +399,19 @@ function addConvTaskPage(scenarioID, description, callback){
     }
 }
 
-
+function addConclusionPage(scenarioID, text, callback){
```

Comment:
The conclusion page should have a prompt for student feedback, but I don't see how that would be defined in just the 'text' field. Is the prompt supposed to be hard-coded in the player frontend?